README.md

```diff
@@ -50,6 +50,16 @@ or if you are on windows, run:
 
     start /wait "" build/Miniforge3-Windows-x86_64.exe /InstallationType=JustMe /RegisterPython=0 /S /D=%UserProfile%\Miniforge3
 
+#### Mambaforge
```

Comment:
Can you move this up to where the other links are

Reply:
Done.